src/pages/trader-tools/common/_utility.js

```diff
@@ -0,0 +1,110 @@
+import validation from './_validation';
+
+const getMargin = (values) => {
+    const { symbol, volume, pointValue, contractSize } = values
+    let margin_formula
+    const STEPINDEX_VALUE = 100
+    const RANGEBREAK100VALUE = 400
+    const RANGEBREAK200VALUE = 800
+
+    if (symbol.name === 'Step Index') {
+        margin_formula = volume * STEPINDEX_VALUE
+    } else if (symbol.name === 'Range Break 100 Index') {
+        margin_formula = volume * RANGEBREAK100VALUE
+    } else if (symbol.name === 'Range Break 200 Index') {
+        margin_formula = volume * RANGEBREAK200VALUE
+    } else {
+        margin_formula = volume * contractSize * pointValue
+    }
+
+    return toFixed(margin_formula)
+}
+
+const toFixed = (val) => {
+    return parseFloat(val.toFixed(3)).toLocaleString()
+}
+
+const resetValidation = (values) => {
+    const errors = {}
+    const symbol_error = validation.symbol(values.symbol)
+    const volume_error = validation.volume(values.volume)
+    const pointValue_error = validation.pointValue(values.pointValue)
+
+    if (symbol_error) {
+        errors.symbol = symbol_error
+    }
+    if (volume_error) {
+        errors.volume = volume_error
+    }
+    if (pointValue_error) {
+        errors.pointValue = pointValue_error
+    }
+
+    return errors
+}
+
+const getMarginCurrency = (symbol) => {
+    let currency = 'USD'
+    if (symbol.market === 'synthetic_indices' || symbol.market === 'commodities') {
+        currency = 'USD'
+    }
```

Comment:
```suggestion
```
Should this be omitted since the default is already 'USD' ?

Reply:
Cool.. Resolved